README.md

```diff
@@ -148,7 +152,8 @@ Support the project by [starring it](https://github.com/VishnuSanal/Quotes/starg
 </pre>
 
 ## 📬 Feedback and Suggestion
-We appreciate your effort to help us in making this app more awesome. Your feedback is immensely helpful to us and we love to hear from you. Leave your feedback/suggestions [here](https://t.me/QuotesStatusCreator) or feel free to create an [issue](https://github.com/VishnuSanal/Quotes/issues/new/choose)! :)
+Helpful advice makes the app better. Your feedback is immensely helpful, and we love to hear from you. \
```

Comment:
Why the carriage return at the end? Please revert if that was a mistake.

Reply:
It is intentional. That way the sentences both start on a newline.

Currently it is
Sentence 1 lorem ipsum. Sentence 2 lorem ipsum.

And I want it to be

Sentence 1 lorem ipsum.
Sentence 2 lorem ipsum.